Code Samples/Fib/Makefile

```diff
@@ -0,0 +1,18 @@
+CXX      := g++
+CXXFLAGS := -std=c++11 -Wall -Wextra -pthread
```

Comment:
Is this supposed to use -pthread still?

Reply:
The issue refers to `Adjust build scripts or CMake/Makefiles to avoid needing -pthread`